client/src/components/Canvas/index.js

```diff
@@ -1,7 +1,7 @@
-import { Canvas } from './Canvas';
+import  Canvas  from './Canvas';
 import { Button } from 'react-bootstrap';
 
-const CanvasView = (props) => {
+const CanvasControls = (props) => {
```

Comment:
I want to discuss the future structure of this component, we might have to split it up later.